ui/glide/src/main/java/de/danoeh/antennapod/ui/glide/ResizingOkHttpStreamFetcher.java

```diff
@@ -45,12 +45,13 @@ public void onDataReady(@Nullable InputStream data) {
                     callback.onDataReady(null);
                     return;
                 }
+                OutputStream outputStream = null;
                 try {
                     tempIn = File.createTempFile("resize_", null);
                     tempOut = File.createTempFile("resize_", null);
-                    OutputStream outputStream = new FileOutputStream(tempIn);
+                    outputStream = new FileOutputStream(tempIn);
                     IOUtils.copy(data, outputStream);
-                    outputStream.close();
+                    //outputStream.close();
```

Comment:
I think if we don't close it here, we risk an incomplete file when reading it a few statements later

Reply:
yes，it‘s right.
but it will be closed in finally block.
if we colose here, there maybe throws an exception on before statement "IOUtils.copy(data, outputStream);".
closing it on the finally block,it will be execute in anyway.

Reply:
The finally block is too late. It needs to be closed earlier. By the time that the finally block is executed, we have already tried to read the created (incomplete) file.